sdk/metricsadvisor/Azure.AI.MetricsAdvisor/src/Models/Hook/AlertingHook.cs

```diff
@@ -8,6 +8,8 @@
 namespace Azure.AI.MetricsAdvisor.Models
 {
     /// <summary>
+    /// An alert notification to be triggered After an anomaly is detected by Metrics Advisor.
+    /// An alert setting can be used with multiple detection configurations, various parameters are available to customize your alert rule.
```

Comment:
This second line doesn't seem like a description of `AlertingHook`. More like `AnomalyAlertConfiguration`.